tests/unit/components/HdIcon.spec.js

```diff
@@ -84,4 +84,21 @@ describe('HdIcon', () => {
     expect(targetedPaths.length).toBeGreaterThan(0);
     expect(targetedPaths.is(`.${TEST_CLASS}`)).toBe(true);
   });
+
+  it('adds title and desc tag to the svg', async () => {
+    const wrapper = wrapperBuilder({
+      props: {
+        src: 'fake/icon4.svg',
+        title: 'This make me accessible.',
+        description: 'Some description.',
+        id: 'icon-id',
+      },
+    });
+
+    await wrapper.vm.$nextTick();
+
+    expect(wrapper.contains('title')).toBeTruthy();
+    expect(wrapper.contains('desc')).toBeTruthy();
+    expect(wrapper.attributes('aria-labelledby')).toBe('icon-id-title icon-id-desc');
```

Comment:
Because this manipulated the dom, and we have to test a few things (tags have been added, tags have the right content, the attributes are added and they have the right value...) I think a snapshot test would be perfect here.